d3a_api_client/redis_device.py

```diff
@@ -75,6 +77,14 @@ def select_aggregator(self, aggregator_uuid, is_blocking=True):
                 return transaction_id
             except AssertionError:
                 raise RedisAPIException(f'API has timed out.')
+        self.pubsub.unsubscribe(
+            [f'{self.area_id}/response/register_participant',
+             f'{self.area_id}/response/unregister_participant',
+             f'{self._channel_prefix}/events/market',
+             f'{self._channel_prefix}/events/tick',
+             f'{self._channel_prefix}/events/trade',
+             f'{self._channel_prefix}/events/finish']
+        )
```

Comment:
Can you please explain why this is needed? Maybe the client wants to see some events even though it as selected an aggregator!?

Reply:
Discussed offline. Please add a flag to toggle this unsubscription. Thanks